spec/ddtrace/contrib/elasticsearch/patcher_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'spec_helper'
```

Comment:
These specs are not running in CI. You need to add these specs to the [appropriate spots in the Rakefile](https://github.com/DataDog/dd-trace-rb/blob/965809a782dc3c5dbb6fcbd4a004db17b663ac2a/Rakefile#L225).